% Relevant Links:
% http://www.nitrc.org/plugins/mwiki/index.php/neurobureau:AthenaPipeline
% http://neurovault.org/collections/1025/

%%
%% Copyright 2007, 2008, 2009 Elsevier Ltd
%%
%% This file is part of the 'Elsarticle Bundle'.
%% ---------------------------------------------
%%
%% It may be distributed under the conditions of the LaTeX Project Public
%% License, either version 1.2 of this license or (at your option) any
%% later version.  The latest version of this license is in
%%    http://www.latex-project.org/lppl.txt
%% and version 1.2 or later is part of all distributions of LaTeX
%% version 1999/12/01 or later.
%%
%% The list of all files belonging to the 'Elsarticle Bundle' is
%% given in the file `manifest.txt'.
%%

%% Template article for Elsevier's document class `elsarticle'
%% with numbered style bibliographic references
%% SP 2008/03/01
%%
%%
%%
%% $Id: elsarticle-template-num.tex 4 2009-10-24 08:22:58Z rishi $
%%
%%
\documentclass[preprint,12pt,5p]{elsarticle}
\usepackage{rotating}


\usepackage{array}
\newcolumntype{L}[1]{>{\raggedright\let\newline\\arraybackslash\hspace{0pt}}p{#1}}
\newcolumntype{R}[1]{>{\raggedleft\let\newline\\arraybackslash\hspace{0pt}}p{#1}}

\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
  
  
%% Use the option review to obtain double line spacing
%% \documentclass[preprint,review,12pt]{elsarticle}

%% Use the options 1p,twocolumn; 3p; 3p,twocolumn; 5p; or 5p,twocolumn
%% for a journal layout:
%% \documentclass[final,1p,times]{elsarticle}
%% \documentclass[final,1p,times,twocolumn]{elsarticle}
%% \documentclass[final,3p,times]{elsarticle}
%% \documentclass[final,3p,times,twocolumn]{elsarticle}
%% \documentclass[final,5p,times]{elsarticle}
%% \documentclass[final,5p,times,twocolumn]{elsarticle}

%% if you use PostScript figures in your article
%% use the graphics package for simple commands
%% \usepackage{graphics}
%% or use the graphicx package for more complicated commands
%% \usepackage{graphicx}
%% or use the epsfig package if you prefer to use the old commands
%% \usepackage{epsfig}

%% The amssymb package provides various useful mathematical symbols
\usepackage{amssymb}
%% The amsthm package provides extended theorem environments
%% \usepackage{amsthm}
\usepackage{url}


%% The lineno packages adds line numbers. Start line numbering with
%% \begin{linenumbers}, end it with \end{linenumbers}. Or switch it on
%% for the whole article with \linenumbers after \end{frontmatter}.
%% \usepackage{lineno}

%% natbib.sty is loaded by default. However, natbib options can be
%% provided with \biboptions{...} command. Following options are
%% valid:

%%   round  -  round parentheses are used (default)
%%   square -  square brackets are used   [option]
%%   curly  -  curly braces are used      {option}
%%   angle  -  angle brackets are used    <option>
%%   semicolon  -  multiple citations separated by semi-colon
%%   colon  - same as semicolon, an earlier confusion
%%   comma  -  separated by comma
%%   numbers-  selects numerical citations
%%   super  -  numerical citations as superscripts
%%   sort   -  sorts multiple citations according to order in ref. list
%%   sort&compress   -  like sort, but also compresses numerical citations
%%   compress - compresses without sorting
%%
%% \biboptions{comma,round}

\biboptions{sort&compress}


\journal{Neuroimage}

\begin{document}



\begin{frontmatter}

\title{The Neuro Bureau ADHD-200 Preprocessed Repository}

% - CC - What about just sticking to the name of the repository? The experiment
% part might make people think that is not an ongoing data sharing initiative.
% We are sharing more that just the statistical derivatives and
% we are also sharing processed structural MRI. I think that capturing all of that
% in a subtitle would be too long.
% - PB - Agreed, it's a cool title. 

% The ADHD-200 Preprocessed experiment: lowering the barriers to a neuroimaging competition
% The ADHD-200 Preprocessed Repository:\Openly sharing preprocessed data and derivatives


\author[label0,label1,label2]{Pierre Bellec\corref{cor1}}
\address[label0]{The Neuro Bureau}
\address[label1]{Centre de Recherche de l'Institut Universitaire de G\'eriatrie de Montr\'eal, Montr\'eal, CA}
\address[label2]{D\'epartement d'Informatique et de Recherche Op\'erationnelle, Universit\'e de Montr\'eal, Montr\'eal, CA}

\cortext[cor1]{Corresponding authors.}

\ead{pierre.bellec@criugm.qc.ca}
\ead[url]{bellec.simexp-lab.org}

\author[label0,label3]{Carlton Chu}
\address[label3]{Deep Mind, London, UK}
\ead{carltonchu1@gmail.com}

\author[label0,label1,label4]{Fran\c{c}ois Chouinard-Descortes}
\address[label4]{McGill University, Montreal, CA}
\ead{carltonchu1@gmail.com}

\author[label0,label5]{Daniel S. Margulies}
\address[label5]{Max Planck Research Group for Neuroanatomy \& Connectivity, Max Planck Institute for Human Cognitive and Brain Sciences, Leipzig, Germany}
\ead{daniel.margulies@gmail.com}

\author[label0,lab6,lab7]{R. Cameron Craddock\corref{cor1}}
\address[lab6]{Computational Neuroimaging Laboratory, Center for Biomedical Imaging and Neuromodulation, Nathan S. Kline Institute for Psychiatric Research, Orangeburg, NY, USA}
\address[lab7]{Center for the Developing Brain, Child Mind Institute, New York, NY, USA}
\ead{ccraddock@nki.rfmh.org}
\ead[url]{computational-neuroimaging-lab.org}

\begin{abstract}
In 2011, the ``ADHD-200 Global Competition'' was held with the aim of identifying biomarkers of attention deficit hyperactivity disorder from resting-state functional magnetic resonance imaging (rs-fMRI) and structural MRI (s-MRI) data collected on 973 individuals. Statisticians and computer scientists were potentially the most qualified for the machine learning aspect of the competition, but generally lacked the specialized skills to implement the necessary steps of data preparation for rs-fMRI. Realizing this barrier to entry, the Neuro Bureau prospectively collaborated with all competitors by preprocessing the data and sharing these results at the Neuroimaging Informatics Tools and Resources Clearinghouse (NITRC) (\url{http://www.nitrc.org/frs/?group_id=383}). This ``ADHD200 preprocessed'' release included multiple analytical pipelines to cater to different philosophies of data analysis. The processed derivatives included denoised and registered 4D fMRI volumes, regional time series extracted from brain parcellations, maps of 10 intrinsic connectivity networks, fractional amplitude of low frequency fluctuation, and regional homogeneity, along with grey matter density maps. The data was used by several teams who competed in the ADHD-200 Global Competition, including the winning entry by a group of biostaticians. To the best of our knowledge, the ADHD200 preprocessed release was the first large public resource of preprocessed resting-state fMRI and structural MRI data, and remains to this day the only resource featuring a battery of alternative processing paths. 
\end{abstract}

\begin{keyword}
%% keywords here, in the form: keyword \sep keyword
%% MSC codes here, in the form: \MSC code \sep code
%% or \MSC[2008] code \sep code (2000 is the default)
preprocessed fMRI, data sharing, neuroimaging competition
\end{keyword}

\end{frontmatter}

%%
%% Start line numbering here if you want
%%
% \linenumbers
\section*{Highlights}
\begin{itemize}
\item Processed derivatives generated from structural and resting-state fMRI in 973 individuals. 
\item The sample includes 585 typically developing children, 362 children with attention deficit hyperactivity disorder, and 26 with unknown diagnostic status.
\item Functional MRI volumes were denoised and aligned using two alternative processing pipelines. 
\item Derivatives include regional time series, grey matter segmentation as well as two variants of connectivity maps and a measure of intrinsic brain activity. 
\item These resources are freely available, with minimal access requirements. 
\end{itemize}
\section{Introduction}

In 2011, the ``ADHD-200 Global Competition'' was held with the aim of engaging researchers from a variety of analytical backgrounds to identify biomarkers of attention deficit hyperactivity disorder (ADHD) from resting-state functional magnetic resonance imaging (rs-fMRI) and structural MRI (s-MRI) data \cite{Milham2012}. The competition made use of the ``ADHD-200 Sample'' data collection that was aggregated from eight independent sites and shared through the Intenational Neuroimaging Datasharing Initiative (INDI) \cite{Mennes2013}. The data includes rs-fMRI, structural MRI (sMRI), and basic phenotypic information for 973 individuals (585 TDC, 362 ADHD, 26 unknown) \cite{Milham2012}. Competitors were given five and a half months to optimize a classification algorithm on training data (776 individuals) and submit their predicted clinical labels for test data for which diagnostic information was withheld. The competition data was distributed in a raw form and, before any analysis could begin, the images had to be preprocessed to make them comparable across individuals and reduce noise. These preprocessing steps present a significant hurdle for would-be competitors who do not have the specialist knowledge of neuroimaging methods or access to high performance computing resources. Realizing this barrier to entry, the Neuro Bureau prospectively collaborated with all competitors by preprocessing the data and sharing these results.

The ``ADHD-200 Preprocessed'' is a repository of preprocessed rs-fMRI and s-MRI data along with statistical derivatives from the ADHD-200 Sample. Rather than favoring a specific processing strategy, we followed a pluralistic approach by preprocessing the data using multiple pipelines (called ``Athena'', ``Burner'' and ``NIAK'') that differed in toolsets used, the philosophy motivating choices of algorithms and parameters, and the statistical derivatives calculated. The Athena pipeline processed rs-fMRI and s-MRI images using a combination of AFNI \cite{cox1996afni} and FSL \cite{smith2004advances} neuroimaging toolkits. The Burner pipeline used SPM8 \cite{ashburner2012spm8} to process s-MRI data for voxel-based morphometry style tools. The NIAK pipeline processed rs-fMRI and s-MRI using a workflow composed of tools from the neuroimaging analysis kit (NIAK) \cite{Bellec2011}, which were assembled with the Pipeline System for Octave and Matlab (PSOM) \cite{Bellec2012}. 

% PB -> CC: the following information is redundant with the rest of the paper. It's also not introductory material. As the paper is short, I would tend to avoid repetition. 

%The results from the pipelines are shared on the Neuro Bureau project page at the Neuroimaging Informatics Tools and Resources Clearinghouse (NITRC)\footnote{\url{http://www.nitrc.org/frs/?group_id=383}}. The data was used by several teams who competed in the ADHD-200 Global Competition, including the winning entry by a group of biostaticians with limited prior neuroimaging experience \cite{Eloyan2012}. The initiative has had an significant impact beyond the competition, having been used in 49 publications \cite{Rangarajan2014, Liang2012, Tabas2014, Rangarajan2015, Mahanand2013, Lifshitz2012, Fujita2013, Ji2011, Li2015, Li2013, Liu2012, DosSantosSiqueira2014, Olivetti2014, Han2015, Wang2013a, Subramanian2013, Dey2014, Bellec2012, Bohland2012, Chang2012, Cheng2012, Colby2012, Dai2012, Dey2012, Eloyan2012, Olivetti2012, Sato2012a, Carmona2015, Carmona2015a, Hou2015, Deshpande2015, She2014, Lavoie-Courchesne2012b, Chen2015, Nunez-Garcia2015, Solmaz2012, Anderson2014, KadkhodaeianBakhtiari2012, Sato2013, Kyeong2015, Sato2012, Takahashi2012, He2013, Kong2013, Yao2013, Yang2015, Ahn2015, Fujita2014, Reiss2014}, three PhD theses \cite{Colby2012a, Dey2013, Zhang2012}, three master's dissertations \cite{VanGalenLast2011, Vidal2014, Wang2013} and one patent \cite{Dey2013} in just over 3 years, with more in press or under review.

\section{Organization and access to the repository}

The ADHD-200 preprocessed data was released in 2011 and can be downloaded from NITRC\footnote{\url{http://www.nitrc.org/frs/?group_id=383}}. No data usage agreement is required to access or download the data, the only requirement is registering for a free NITRC account. This registration enables downloads to be tracked for usage statistics. Registered users can also be contacted in the event that errors would be found. The ADHD-200 Sample allows unrestricted data usage for non-commercial research purposes provided that the specific datasets included in an analysis be cited appropriately and that their funding sources be acknowledged\footnote{\url{http://fcon_1000.projects.nitrc.org/indi/adhd200/}}. There are no more restrictions placed on the preprocessed data or derivatives other than the request that the ADHD-200 Preprocessed Initiative is cited appropriately and that the specific pipeline is acknowledged in publications using the data. A forum is available on the Neuro Bureau's NITRC project page for users to ask questions or report
problems\footnote{\url{http://www.nitrc.org/forum/forum.php?forum_id=2046}}. Questions regarding data acquisition or phenotypic variables should be directed to INDI's support forum\footnote{\url{http://www.nitrc.org/forum/forum.php?forum_id=1735}}. 


\section{Contents of the repository}

The ADHD-200 Preprocessed repository contains preprocessed outputs and derivatives for data from the ADHD-200 Sample, which includes 973 individuals (352 F) between the ages of 7 and 27 aggregated from 17 different studies conducted across 8 different sites (for a breakdown of age and sex by diagnosis see table \ref{part_table}). For each individual, phenotypic data includes sex, age, handedness, ADHD diagnosis (585 TDC, 362 ADHD, 26 unknown), ADHD subtype (ADHD-combined, ADHD-inattentive, ADHD-hyperactive/impulsive), one of three different measures of ADHD severity, one of five measures of intelligence, co-morbid diagnoses, and whether or not they have used medication to treat their symptoms \cite{Milham2012}. Imaging data for each individual includes one or more T1-weighted high-resolution s-MRI scan (s-MRI) and one or more rs-fMRI scan. Most of the data was collected from a single imaging session, although a second session is available for 15 individuals from the WUSTL site. There is a substantial amount of variation in how the data was collected across sites including the type of MRI system and scanning parameters used, the length of the rs-fMRI scans, and the instructions given to the participants prior to the scan (see tables \ref{smri_params_table} and \ref{rsfmri_params_table}). 

Nearly all of the imaging data from the ADHD-200 Sample was included in the preprocessing effort, some individuals were excluded for poor quality or missing data. The results of the preprocessing are made available as a collection of compressed tar files that are organized by pipeline, sites of data collection, training and test samples, as well as by derivative. A group level file containing the phenotypic data is available in comma-separated-values format (.csv). 

Shared preprocessed data and extracted features include:
\begin{itemize}
\tightlist
\item 3D grey matter density maps suitable for voxel-based morphometry as compressed nifti - Athena, Burner and NIAK
\item 4D preprocessed resting-state fMRI data including limited intermediaries and quality assessment (Athena and NIAK)
\item Average time courses for brain regions from structurally defined atlases (Athena)
\item Average time courses for brain regions for regions defined by functional parcellation (Athena and NIAK)
\item Spatial maps for 10 intrinsic connectivity networks (ICNs), fractional amplitude of low frequency fluctuations (fALFF), and regional homogeneity (ReHo) - Athena
\end{enumerate}

% PB->CC I did the breakdown by derivatives, not by pipeline. Otherwise there is too much repetition between NIAK and Athena. This is true for content description but even more for the methods section, where it's nice for example to have the different approaches for generating parcellations under one section, rather than disjointed into two "Athena" and "NIAK" sections. I've also left the details of the methods (e.g. resolution for resampling) to the methods section. 

% I understand, but I am not sure that the repetition is bad. From my understanding, since this is essentially a data paper, we need to be very explicit about what data is available. the way it was before missed alot of useful details, i haven't removed your original content yet, just experimenting with an alternative approach. I would actually prefer it if we could put it into a table, but since what is available varies so much by pipeline, it would be a very lopsided table. Also, since most of the derivatives are pipeline specific, you are essentially breaking them down by derivatives. I think it would be better if the processing and 'contents' sections were merged.


\begin{figure}[!t]
\begin{center}
  \includegraphics[width=0.5\textwidth]{Figure_01}
  \caption{Derivatives from the Athena pipeline including ten intrinsic connectivity networks (ICNs), fractional amplitude of low-frequency fluctuations (fALFF) and regional homogeneity (ReHo). Asterisks (*) in the latter two derivatives denote the difference in colorbar max/min, as indicated.}
  \label{fig_derivatives}
\end{center}
\end{figure}


\subsection{Athena Pipeline} 
The Athena pipeline\footnote{\url{http://www.nitrc.org/plugins/mwiki/index.php/neurobureau:AthenaPipeline}}  processed rs-fMRI and s-MRI images using a custom BASH script that combined AFNI \cite{cox1996afni} and FSL \cite{smith2004advances} neuroimaging toolkits. The processing scripts for each site are distributed in the repository, along with output log files for each processed dataset.

\subsubsection{Structural processing} Athena's s-MRI pipeline began with skull-stripping the images to remove non-brain tissue and background from the images \cite{smith2002_bet} and segmenting the results into white matter (WM), cerebro-spinal fluid (CSF), and grey matter (GM) probability maps \cite{zhang2001_fast}. A non-linear warp was calculated between the skull-off image and MNI space as represented by the NIHPD 4.5\textendash18.5y age-specific assymmetric template \cite{fonov2011unbiased} using a two step procedure that calculates a linear transform \cite{jenkinson2002_flirt} that is subsequently refined by a non-linear registration procedure \cite{andersson2007non}. \emph{Shared s-MRI outputs include:} skull stripped whole brain images and smoothed (by a 6-mm FWHM Gaussian) and unsmoothed GM density maps in MNI space at 1$\times$1$\times$1 mm$^3$ resolution, along with the FSL fNIRT non-linear warp, as compressed nifti files (.nii.gz). 

\subsubsection{Functional processing} Athena's rs-fMRI pipeline involved removing the first four volumes to allow for magnetization to reach equilibrium, site-specific slice timing correction to the middle slice, re-aligning each volume to the first volume to correct for motion \cite{cox1999_motion}, and calculating a linear transform between the mean functional volume and the corresponding s-MRI \cite{jenkinson2002_flirt}. The rs-fMRI to s-MRI transform was then combined with the s-MRI to MNI non-linear warp to write the functional into MNI152 space at $4\times4\times4$ mm$^3$ resolution. Mean WM and CSF signals extracted using the masks calculated during s-MRI processing were included along with 6 head motion parameters and a third-order polynomial in voxel-wise nuisance regression models to remove variation due to physiological noise, head motion and scanner drifts from the time series\cite{lund2006_nvr,fox2005}. The resulting de-noised time series were band-pass filtered ($0.009$ Hz $< f < 0.08$ Hz) to limit the data to the frequencies implicated in resting state functional connectivity \cite{biswal1995,cordes2001} and then spatially smoothed with a 6-mm FWHM Gaussian filter. \emph{Shared rs-fMRI outputs include:} denoised rs-fMRI volumes, with and without temporal bandpass filtering, in MNI space (compressed 4D niftis, nii.gz), the mean rs-fMRI image and brain mask in template space (nii.gz), and six parameter head motion traces (tab separated value, AFNI .1D files). 

\paragraph{Time series for structurally defined brain areas} Regional time series were extracted for the automated anatomical labeling (AAL) \cite{tzourio2002automated}, Eickhoff-Zilles (EZ) \cite{eickhoff2005new}, Harvard-Oxford (HO) \cite{HO_atlas_1, HO_atlas_2, HO_atlas_3, HO_atlas_4}, and Talairach and Tournoux (TT) \cite{lancaster2000automated} parcellations. The EZ atlas was derived from the max-propagation atlas distributed with the SPM Anatomy Toolbox\footnote{\url{http://www.fz-juelich.de/inm/inm-1/EN/Forschung/_docs/SPMAnatomyToolbox/SPMAnatomyToolbox_node.html}} and was transformed into template space using the Colin 27 template (also distributed with the toolbox) as an intermediary. The HO atlas was constructed from 25\% thresholded cortical and subcortical max-propagation atlases distributed with FSL. The atlases were bisected into left and right hemispheres at the midline ($x=0$), ROIs representing left/right WM, left/right GM, left/right CSF and brainstem were removed from the subcortical atlas and then the subcortical and cortical ROIs were combined into a single atlas. The AAL atlas distributed with the SPM8 version of the AAL Toolbox\footnote{\url{http://www.cyceron.fr/index.php/en/plateforme-en/freeware}} and the TT atlas distributed with AFNI were coregistered and warped into template space. Each of the anatomical templates were resampled into the functional space using nearest-neighbor interpolation. Time series were extracted for each atlas from both the filtered and unfiltered data by averaging the voxel time series contained within each labeled region and are distributed in tab-separated values format (AFNI .1D). Each of the conformed ROI atlases are available as compressed 3D nifti files (.nii.gz).

\paragraph{Time series for functionally defined regions} The CC200 and CC400 functional brain parcellations were constructed using a two-stage spatially-constrained spectral clustering procedure \cite{craddock2012whole} applied to unfiltered preprocessed rs-fMRI data from a subset ($N=650$) of the participants in the training dataset. Participants were chose for inclusion based on registration quality and after exclusion of participants with more than 3 mm translation or 3 degrees rotations in their motion parameters. To reduce computation time, the clustering was restricted to grey matter using a group GM mask that was constructed by averaging individual GM masks derived from freesurfer automated segmentation. Although 200 and 400 ROIs were specified in the functional parcellation procedure, the normalized cut algorithm resulted in 190 and 351 clusters respectfully. Time series were extracted for each atlas from both the filtered and unfiltered data by averaging the voxel time series contained within each labeled region and are distributed in tab-separated values format (AFNI .1D). CC200 and CC400 brain parcellations are available as compressed 3D nifti files (.nii.gz).

\paragraph{ICN time series and spatial maps} Time series and spatial maps were derived for 10 ICNs that have been found to be consistent across resting state and a variety of neuroimaging tasks  \cite{smith2009correspondence} by applying a modified dual regression approach \cite{Beckmann2009dualreg} to the unfiltered preprocessed data. A spatial multiple regression was first used to extract time courses corresponding to each network. In a second step, each time course was independently correlated with whole brain time series to generate subject-specific functional connectivity maps for each network. Alternatively, all time courses were entered simultaneously into a multiple (temporal) regression, and the regression cofficients associated with each time course constituted the functional connectivity maps. The resulting ICN time series are distributed as tab-separated values (AFNI .1D) files and the spatial maps for both temporal regression approaches are distributed as compressed 4D NIFTI files (.nii.gz).

\paragraph{fALFF and ReHo} Whole brain fALFF maps were generated by dividing the variance of each voxel's bandpass-filtered time series by the variance of its unfiltered time series \cite{zuo2008falff}. ReHo was estimated from the unfiltered data at each voxel by the Kendall's Coefficient of Concordance \cite{kendall1939w} between the voxel and its 26 face-, edge-, and corner- touching neighbors. The resulting fALFF and ReHo whole brain maps are distributed as compressed 3D NIFTI files (.nii.gz).

\subsubsection{Quality Assessment} 

No specific quality control was applied to Athena data before release, this was left out to accommodate different opinions on what qualifies as good data. In addition to files containing the six motion parameters for each rs-fMRI dataset, and anatomical and mean EPI images in template space, which can be visually inspected to determine high motion data or poor registrations, quality metrics derived from structural data, GM masks, mean EPI images, fALFF maps and FC maps were included to help with the QC process. For each data type a mean and standard deviation image is calculated from all of the scans from all of the subjects. These images are used to perform a voxel-wise z-score transformation on each data type for each subject $z_i = \frac{(v_i-m_i)}{\sigma_i}$. The absolute values of these z-scores are then thresholded at 3, and then summed across voxels for a quality score for that image. The higher the resulting sum, the larger the number of voxels a image has that are $\geq 3$ and the more likely the images are outliers. Using this metric it is possible to rank images, and allows a more directed search for poor quality scans. These metrics are distributed in participant specific text files along with the BASH scripts that implement the procedure.

\subsection{Burner Pipeline}

The Burner pipeline\footnote{\url{http://www.nitrc.org/plugins/mwiki/index.php/neurobureau:BurnerPipeline}} used SPM8 \cite{ashburner2012spm8} to process s-MRI data for voxel-based morphometry \cite{ashburner2000vbm} style analyses.

\subsubsection{Structural processing} 

Processing begins by segmenting s-MRI images into GM and WM probability maps using SPM8's unified segmentation procedure, which iteratively registers the data to a template and performs tissue classification until both are optimized \cite{ashburner2005unified}. Next, SPM8's DARTEL toolbox \cite{ashburner2007dartel} was used to generate a template from the s-MRI data of all participants using another iterative method based on that calculates the nonlinear transforms for warping all participant WM and GM maps to a common space. The resulting non-linear deformations were applied to each participants GM probability maps to transform them into the space of population average at 1.5 $\times$ 1.5 $\times$ 1.5 mm$^3$ resolution and modulated to conserve the global tissue volumes after normalization. The resulting grey matter density maps are distributed at 3D NIFTI files (.nii).

\subsubsection{Quality Assessment}
Stringent quality control was not applied to the data to accommodate different opinions on what constitutes poor quality data. Images for four participants were excluded after visual inspection by Dr. Chiu because they were determined to be too poor quality to be processed.

\subsection{NIAK Pipeline}

Both the Athena and NIAK pipelines released denoised rs-fMRI volumes resampled into a reference, stereotaxic space (variants of the MNI template) as compressed 4D nifti files (.nii.gz). Transformations from native to stereotaxic space for each dataset were released in separate files, in FSL wrap .nii.gz file for Athena and in .xfm/.mnc MINC wrap files for NIAK. In addition, ancillary files such as the estimated rigid-body motion parameters (AFNI .1D tab-separated-value for Athena, and HDF5 files .mat files for NIAK), mean functional image (.nii.gz files) as well as individual and group brain masks (.nii.gz files) were released in separate archives. 

\subsection{Regional time series} Regional time series were extracted based on a series of different brain parcellations. The NIAK pipeline only used functional parcels generated by a region growing method \cite{bellec2006identification} with two resolutions: 954 and 2843 parcels. The  NIAK distributed them as HDF5 files (.mat Matlab). All brain parcellations were also released as compressed nifti files (.nii.gz). 

\subsection{Brain maps}
The Athena pipeline derived a number of brain maps derived from the preprocessed fMRI datasets: fractional amplitude of low frequency fluctuations (fALFF) maps, regional homogeneity (ReHO) maps and dual-regression based on a group independent component analysis (ICA) with 10 resting-state networks \citep{smith2009correspondence}. One compressed nifti file (.nii.gz) was released per fMRI dataset and type of brain map, with in addition the time courses of the dual regression available in text files (.1D AFNI).



\begin{table}[!ht]
\caption{{\bf ADHD-200 participants by site.} BHBU: Bradley Hospital/ Brown University, KKI: Kennedy Krieger Institute, NI: NeuroIMAGE sample, NYU: New York University Child Study Center, OHSU: Oregon Health Sciences University, PKU: Peking University, Pitt: University of Pittsburgh, WUSTL: Washington University at Saint Louis, avg.: average. $^*$Diagnostic labels are current not available for BHBU, they have been listed as TDC in the table, but not included in the totals.}\label{part_table}
  \begin{tabular}{llrrrr}
      \hline
       & & \multicolumn{2}{c}{TDC} & \multicolumn{2}{c}{ADHD}\\
      Site & Sex & N & Age Range (avg.) & N & Age Range (avg.) \\
        \hline
    \noalign{\vskip 1ex}  
    BHBU & F & 17$^*$ & 8 - 18 (13.8) 
             & 0 & - \\
         & M &  9$^*$ & 12.00 - 17.83 (15.96) 
             & 0 & - \\
    KKI  & F & 28 & 8.12 - 12.36 (10.19) 
             & 10 & 8.10 - 12.99 (9.97) \\
         & M & 41 & 8.02 - 12.87 (10.35) 
             & 15 & 8.19 - 12.98 (10.08) \\
    NI  & F & 25 & 11.66 - 26.31 (18.69) 
             & 5 & 11.85 - 19.66 (15.07) \\
         & M & 12 & 12.84 - 25.04 (18.02) 
             & 31 & 11.05 - 20.89 (17.05) \\
    NYU  & F & 55 & 7.17 - 17.96 (12.20) 
             & 34 & 7.35 - 17.15 (10.27) \\
         & M & 56 & 7.19 - 17.96 (12.20) 
             & 117 & 7.24 - 17.61 (11.19) \\
    OHSU  & F & 40 & 7.33 - 12.50 (8.98) 
             & 13 & 7.42 - 11.33 (9.01) \\
         & M & 30 & 7.17 - 11.92 (9.48) 
             & 30 & 7.42 - 11.83 (8.94) \\
    PKU  & F & 59 & 8.33 - 15.17 (10.90) 
             & 10 & 8.83 - 15.92 (10.88) \\
         & M & 84 & 8.08 - 14.92 (11.80)  
             & 92 & 8.33 - 17.33 (12.22) \\ 
    Pitt  & F & 44 & 10.16 - 20.45 (15.65) 
             & 1 & 15.04 \\
         & M & 50 & 10.11 - 18.96 (14.55)  
             & 3 & 13.72 - 17.03 (15.55) \\ 
    WUSTL  & F & 28 & 7.09 - 21.81 (11.33) 
             & 0 & - \\
         & M & 33 & 7.25 - 21.83 (11.58)  
             & 0 & - \\ 
    \bf{Totals} & F & 279$^*$ & 7.09 - 26.31 (11.9)
               & 73 & 7.35 - 19.66 (10.48) \\
           & M & 306$^*$ & 7.17 - 25.04 (12.20)
               & 288 & 7.24 - 20.89 (11.90) \\
    \noalign{\vskip 1ex}  
    \hline
	\end{tabular}
\end{table}


\begin{sidewaystable}[!ht]
\caption{{\bf Structural MRI acquisition parameters by site.} Seq: imaging sequence, FA: flip angle, TE: echo time, TR: repetition time, TI: inversion recovery delay, PA: parallel acquisition, Res: voxel resolution, BHBU: Bradley Hospital/ Brown University, KKI: Kennedy Krieger Institute, NI: NeuroIMAGE sample, NYU: New York University Child Study Center, OHSU: Oregon Health Sciences University, PKU: Peking University, Pitt: University of Pittsburgh, WUSTL: Washington University at Saint Louis, Trio: Siemens TIM Trio 3T, Allegra: Siemens Allegra, Avanto: Siemens Avanto, MPRAGE: magnetization prepared rapid gradient echo, S: sensitivity encoding (SENSE), G: generalized auto-calibrating partially parallel acquisition (GRAPPA)}\label{smri_params_table}
  \begin{tabular}{lllllllll}
      \hline
    Site & Scanner & Seq & FA & TE & TR & TI & PA & Res. \\
    \hline
    \noalign{\vskip 1ex}  
    BHBU & Trio 3T & 3D MPRAGE & 9$^{\circ}$ & 2.98 ms & 2250 ms & 900 ms & None & $1.00\times1.00\times1.00$ mm$^3$ \\
    KKI & Phillips 3T & 3D MPRAGE & 8$^{\circ}$ & 3.7 ms & 3500 ms & 1000 ms & S $\times$2 & $1.00\times1.00\times1.00$ mm$^3$ \\
    NI & Avanto 1.5T & 3D MPRAGE & 7$^{\circ}$ & 2.95 ms & 2730 ms & 1000 ms & G $\times$2 & $1.00\times1.00\times1.00$ mm$^3$ \\
    NYU & Allegra 3T & 3D MPRAGE & 7$^{\circ}$ & 3.25 ms & 2530 ms & 1100 ms & None & $1.30\times1.00\times1.30$ mm$^3$ \\
    OHSU & Trio 3T & 3D MPRAGE & 10$^{\circ}$ & 3.58 ms & 2300 ms & 900 ms & None & $1.00\times1.00\times1.10$ mm$^3$ \\
    PKU 1 & Trio 3T & 3D MPRAGE & 7$^{\circ}$ & 3.39 ms & 2530 ms & 1100 ms & None & $1.30\times1.00\times1.30$ mm$^3$ \\
    PKU 2 & Trio 3T & 3D MPRAGE & 7$^{\circ}$ & 3.45 ms & 2530 ms & 1100 ms & None & $1.00\times1.00\times1.00$ mm$^3$ \\
    PKU 3 (1) & Trio 3T & 3D MPRAGE & 12$^{\circ}$ & 3.67 ms & 2000 ms & 1100 ms & None & $0.94\times0.94\times1.00$ mm$^3$ \\
    PKU 3 (2) & Trio 3T & 3D MPRAGE & 10$^{\circ}$ & 2.60 ms & 1950 ms & 900 ms & None & $1.00\times1.00\times1.30$ mm$^3$ \\
    PKU 3 (3) & Trio 3T & 3D MPRAGE  & 7$^{\circ}$ & 3.37 ms & 2530 ms & 1100 ms & None & $1.00\times1.00\times1.33$ mm$^3$ \\
    PKU 3 (4) & Trio 3T & 3D MPRAGE & 12$^{\circ}$ & 3.92 ms & 1770 ms & 1100 ms & None & $0.50\times0.50\times1.00$ mm$^3$ \\
    PKU 3 (5) & Trio 3T & 3D MPRAGE & 8$^{\circ}$ & 2.89 ms & 845 ms & 600 ms & None & $1.02\times1.02\times1.30$ mm$^3$ \\
    Pitt & Trio 3T & 3D MPRAGE & 8$^{\circ}$ & 3.43 ms & 2100 ms & 1050 ms & None & $1.00\times1.00\times1.00$ mm$^3$ \\
    WUSTL & Trio 3T & 3D MPRAGE & 8$^{\circ}$ & 3.08 ms & 2400 ms & 1000 ms & G $\times$2 & $1.00\times1.00\times1.00$ mm$^3$ \\
    \noalign{\vskip 1ex}  
    \hline
	\end{tabular}
\end{sidewaystable}

\begin{sidewaystable}[!ht]
\caption{{\bf Resting state fMRI acquisition parameters by site.} Seq: imaging sequence, FA: flip angle, TE: echo time, TR: repetition time, PA: parallel acquisition, N$_{slc}$: number of slices, Th.: slice thickness, Slc. Acq.: slice acquisition order, N$_{TR}$: number of measurements (TRs),  BHBU: Bradley Hospital/ Brown University, KKI: Kennedy Krieger Institute, NI: NeuroIMAGE sample, NYU: New York University Child Study Center, OHSU: Oregon Health Sciences University, PKU: Peking University, Pitt: University of Pittsburgh, Pitt 2: U. Pitt. parameters used for acquiring the testing data, WUSTL: Washington University at Saint Louis, EPI: echo planar imaging, PACE: Prospective Acquisition CorrEction (EPI with prospective motion correction), S: sensitivity encoding (SENSE), G: generalized autocalibrating partially parallel acquisition (GRAPPA), int+: slices were acquired interleaved ascending, seq+: slices were acquired sequentially ascending, var.: the number of measurements varies across datasets, fixate: participants were asked to keep their eyes open and fixate on an image, closed: participants were asked to keep their eyes closed, open: participants were asked to keep their eyes open.}\label{rsfmri_params_table}
  \begin{tabular}{llllllllllll}
    \hline
    Site & Seq & FA & TE & TR & PA & N$_{slc}$ & Th. & Slc. Acq. & Resolution & N$_{TR}$ & Instructions  \\ 
        \hline
    \noalign{\vskip 1ex} 
    BHBU & PACE & 90$^{\circ}$ & 25 ms & 2000 ms  & None & 35 & 3 mm & int+ & $3.0 \times 3.0$ mm$^{2}$ & 256 & fixate \\
    KKI & EPI & 75$^{\circ}$ & 30 ms & 2500 ms  & S $\times3$ & 47 & 3 mm & seq+ & $3.0 \times 3.0$ mm$^{2}$ & 128 & fixate \\
    NI & EPI & 80$^{\circ}$ & 40 ms & 1960 ms  & G $\times2$ & 37 & 3 mm & int+ & $3.5 \times 3.5$ mm$^{2}$ & 266 & eyes closed \\
    NYU & EPI & 90$^{\circ}$ & 15 ms & 2000 ms  & None & 33 & 4 mm & int+ & $3.0 \times 3.0$ mm$^{2}$ & 180 & eyes closed \\
    OHSU & EPI & 90$^{\circ}$ & 30 ms & 2500 ms  & None & 36 & 3.8 mm & int+ & $3.8 \times 3.8$ mm$^{2}$ & 82 & fixate \\
    PKU 1 & EPI & 90$^{\circ}$ & 30 ms & 2000 ms  & None & 33 & 4.2 mm & int+ & $3.1 \times 3.1$ mm$^{2}$ & 240 & closed or fixate \\
    PKU 2 & EPI & 90$^{\circ}$ & 30 ms & 2000 ms  & None & 33 & 3.6 mm & int+ & $3.1 \times 3.1$ mm$^{2}$ & 240 & closed or fixate \\
    PKU 3 & EPI & 90$^{\circ}$ & 30 ms & 2000 ms  & None & 30 & 4.5 mm & int+ & $3.44 \times 3.44$ mm$^{2}$ & 240 & closed or fixate \\
    Pitt & EPI & 70$^{\circ}$ & 29 ms & 1500 ms  & G $\times2$ & 29 & 4.0 mm & int+ & $3.1 \times 3.1$ mm$^{2}$ & 200 & open or closed \\
    Pitt 2 & EPI & 90$^{\circ}$ & 30 ms & 3000 ms  & None & 46 & 3.5 mm & int+ & $3.8 \times 3.8$ mm$^{2}$ & 128 & open or closed \\
    WUSTL & EPI & 90$^{\circ}$ & 27 ms & 2500 ms  & None & 32 & 4.0 mm & int+ & $4.0 \times 4.0$ mm$^{2}$ & var. & fixate \\
    \noalign{\vskip 1ex} 
    \hline

	\end{tabular}
\end{sidewaystable}

\section{Processing strategies}
Further details on the processing strategies can be found on the ``ADHD200 preprocessed'' website\footnote{\url{http://www.nitrc.org/plugins/mwiki/index.php/neurobureau:ADHDpreproc}}.

\subsection{Preprocessed sMRI and rs-fMRI}

\paragraph{The NIAK pipeline} The preprocessing steps in NIAK followed those of the Athena pipeline with differences at the following steps. (1) Only the first three volumes were suppressed, instead of four. (4) Although both Athena and NIAK used the MNI non-linear stereotaxic template space described in \citep{fonov2011unbiased}, the NIAK used the young adult (ICBM152) symmetric variant, while Athena used the NIHPD asymmetric pediatric template (4.5 y.o.-18.5 y.o.). (5) The fMRI volumes were resampled in stereotaxic space at 3 mm isotropic (instead of 4 mm), and this was done only right before spatial smoothing (step 5 was moved before step 8). (6) No regression of confounds was performed. Instead, a physiological noise correction based on automatic labeling of ICA components was implemented \citep{perlbarg2007corsica} after temporal filtering (steps 6 and 7 were inverted). (7) Only a high-pass filter was implemented ($0.01$ Hz $< f$), instead of a band pass. 

\subsection{Regional time series}
Table \ref{tab_parcellations} summarizes the characteristics of the parcellations, which were also presented in Figure \ref{fig_parcellations}. 
\begin{table}[htbp]
\label{tab_parcellations}
\caption{Summary of the characteristics of all brain parcellations used in the ADHD200 preprocessed release. All reported size for the parcels are measured in mm$^3$.}
\begin{tabular}{llrrrrr}
Name & Type & \# parcels & mean size & std size & min size & max size \\
\hline
AAL	    & anatomical &  116 &  16726 & 11896 &  768 & 55552\\
EZ	    & anatomical &  116 &  15880 & 11059 & 1344 & 52608\\
HO	    & anatomical &  111 &  14540 & 15342 &   64 & 99200\\
TT	    & anatomical & 	 97 &  17106 & 16164 &   64 & 70400\\
CC200   & functional &  190 &  11351 &  2001 & 2880 & 17856\\
CC400   & functional &  351 &   6144 &  1207 &   64 & 10048\\
ROI1000 & functional &  954 &   1404 &   366 &   27 & 2781\\
ROI3000	& functional & 2843	&    471 &   109 &   27 & 1026\\
\end{tabular}
\end{table}

% PB - @CC: I found a mismatch between the # of ROIs for AAL, EZ, HO and TT from my own extraction and the NITRC table. Any idea why?
% I suspect you eliminated small regions (HO, TT and CC400 have regions with only one voxel). But I am deriving the stats from the nifti files 
% shipped with the release so I also want to be consistent with what people get. 

\paragraph{Anatomical parcellations} For both Athena and NIAK, regional time series were extracted by averaging preprocessed resting-state volumes within different brain parcels. The Athena pipeline first used a series of parcellations delineated on the basis of anatomical landmarks:


\paragraph{The ROI1000 and ROI3000 functional parcellations} A region-growing algorithm \citep{bellec2006identification} based on the iterative merging of mutual-nearest-neighbours was implemented to generate functional brain parcellations. The spatial dimension was selected arbitrarily by setting the size where the growing process stopped, measured in mm$^3$. Two parameters (1000 mm$^3$ and 330 mm$^3$) were selected, resulting into roughly 1000 and 3000 ROIs covering the grey matter. The regions were built to maximize the average correlation between the time series associated with any pair of voxels of the region. The region growing was applied on the time series concatenated across all subjects (after correction to zero mean and unit variance) at the KKI site (training data only), such that the homogeneity was maximized on average for all subjects, and the small homogeneous regions were identical for all subjects. Because of the temporal concatenation of time series, we had to limit the memory demand, and the region-growing was thus applied independently in each of the 116 areas of the AAL template \citep{tzourio2002automated}. 


% The scripts used to calculate ReHo were generously provided by Dr. Xinian Zuo.


%Details on the NIAK pipeline can be found on the NITRC website.

% http://www.nitrc.org/plugins/mwiki/index.php/niak:FmriPreprocessing064#Summary_of_the_pipeline

\section{Quality control}

Both the Athena and NIAK pipelines generated the maximum change in translation and rotation for the rigid motion parameters estimated for each rs-fMRI dataset. 

 


The output of the Burner pipeline were visually inspected by Dr Chu. The individual results of the NIAK pipeline were visually reviewed by M. Chouinard-Decortes for quality of the registration of fMRI and sMRI data as well as registration of sMRI data in stereotaxic space. The conclusions of the NIAK quality control procedure are available on NITRC\footnote{\url{http://www.nitrc.org/plugins/mwiki/index.php/neurobureau:NIAKPipeline#Quality_control_of_the_preprocessing_-_Training_dataset}}.

\section{Lessons learned and future work}

The ADHD200 preprocessed initiative was a success in terms of its primary objective: the data repository was effectively used by many researchers during and after the ADHD200 competition, with over 10,500 downloads by more than 600 users, as well as 49 publications \cite{Rangarajan2014, Liang2012, Tabas2014, Rangarajan2015, Mahanand2013, Lifshitz2012, Fujita2013, Ji2011, Li2015, Li2013, Liu2012, DosSantosSiqueira2014, Olivetti2014, Han2015, Wang2013a, Subramanian2013, Dey2014, Bellec2012, Bohland2012, Chang2012, Cheng2012, Colby2012, Dai2012, Dey2012, Eloyan2012, Olivetti2012, Sato2012a, Carmona2015, Carmona2015a, Hou2015, Deshpande2015, She2014, Lavoie-Courchesne2012b, Chen2015, Nunez-Garcia2015, Solmaz2012, Anderson2014, KadkhodaeianBakhtiari2012, Sato2013, Kyeong2015, Sato2012, Takahashi2012, He2013, Kong2013, Yao2013, Yang2015, Ahn2015, Fujita2014, Reiss2014}, three PhD theses \cite{Colby2012a, Dey2013, Zhang2012}, three master's dissertations \cite{VanGalenLast2011, Vidal2014, Wang2013} and one patent \cite{Dey2013} derived from the release in just over 3 years, with more in press or under review. The resource was also used outside of the neuroimaging community, with several publications in engineering and statistics journals that do not routinely feature neuroimaging applications \citep[e.g.][]{Liang2012, Rangarajan2015, Rangarajan2014, Mahanand2013, Ji2011, Li2015, Li2013, Liu2012, Subramanian2013, Hou2015, Deshpande2015, She2014, Chen2015, He2013, Kong2013, Yang2015, Ahn2015}. In particular, the winning team of the ADHD200 global competition was based at the Johns Hopkins Biostatistics Department and used ADHD200 
% i don't think that we can conclude that the availability of the time series was the driving factor behind the pubs by non-neuroimagers
% PB->CC OK so I quickly checked some of these references, and they are definitely using a variety of derivatives. OK. 
% Just a note is She2014. In this paper if you blink you will miss the fMRI section. It's 10 lines long, in an otherwise highly theoretical paper, and the other application (also 10 lines) is about yeast gene regulation network. I think this is awesome, and possible because you can grab time series without having any clue what they represent and run with it. But I am extrapolating too much from this example. 
preprocessed to develop their diagnostic algorithm \cite{Eloyan2012}.
\par 
Our conclusions are more mitigated regarding the secondary objective of ``ADHD200 preprocessed'', which was to cater to multiple data processing philosophies. The release was a success in the sense that almost all variants of processing strategies included in the ADHD200 release have been used in follow-up published manuscripts. However, no publication we are aware of actually implemented a face-to-face comparison between processing techniques. With the approach we took, any direct comparison between Athena and NIAK was difficult to interpret, as the two pipelines did not use the same specific stereotaxic space, varied in many ways in terms of selected parameters, and also had quite different approaches for QC, derivatives or even packaging the derivatives. This lack of harmonization was essentially driven by the tight deadline for data release imposed by the competition, rather than a design choice. We since came to the conclusion that it would be useful to build the processing pipelines on a common stereotaxic space and possibly sets of registration, so that the strategies of noise reduction can be more readily compared. 
\par 
The impact of the ``ADHD200 preprocessed'' repository emphasized the need to reduce computational barriers to participation in discovery neuroscience, including but not limited to machine learning competitions based on neuroimaging data. This need is further emphasized by a recent survey of FCP/INDI users in which 70\% of the respondents indicated that the availability of statistical derivatives is either very important or absolutely critical to the neuroimaging community. We do not plan to expand or update the ADHD200 release as such, which we hope will continue to serve as a legacy benchmark dataset. We still believe that a much larger-scale effort will be necessary to unlock the full potential of openly shared neuroimaging data to accelerate neuroimaging research. We are now working on new initiatives learning from the ADHD200 preprocessed experiment, notably including a larger number of alternative analytical workflows as well as a careful harmonization of processing, quality control and data packaging strategies. Our hope is that ADHD200 preprocessed and future related efforts will critically help fMRI researchers to identify optimal analytical paths for a given task. 


%% References
%%
%% Following citation commands can be used in the body text:
%% Usage of \cite is as follows:
%%   \cite{key}         ==>>  [#]
%%   \cite[chap. 2]{key} ==>> [#, chap. 2]
%%

%% References with bibTeX database:

\bibliographystyle{elsarticle-num-names}
% \bibliographystyle{elsarticle-harv}
% \bibliographystyle{elsarticle-num-names}
% \bibliographystyle{model1a-num-names}
% \bibliographystyle{model1b-num-names}
% \bibliographystyle{model1c-num-names}
% \bibliographystyle{model1-num-names}
% \bibliographystyle{model2-names}
% \bibliographystyle{model3a-num-names}
% \bibliographystyle{model3-num-names}
% \bibliographystyle{model4-names}
% \bibliographystyle{model5-names}
% \bibliographystyle{model6-num-names}

\bibliography{adhd200_pubs}


\end{document}

%%
%% End of file `elsarticle-template-num.tex'.
